contracts/base/Escrow.sol

```diff
@@ -104,8 +156,24 @@ contract Escrow is EscrowInterface, TInitializable, Ownable, BaseUpgradeable, Ba
         TInitializable._initialize();
 
         // Initialize tokens list with the borrowed token.
-        // _tokenUpdated(loans.lendingToken()); // TODO Verify the _tokenUpdated function works with the AddressLib
+        // TODO: test balance
+        require(_balanceOf(loans.lendingToken()) > 0, "LENDING_TOKEN_BALANCE_ZERO");
+        _tokenUpdated(loans.lendingToken());
     }
 
     /** Internal Functions */
+
+    function _valueOfIn(address baseAddress, address quoteAddress, uint256 baseAmount) internal view returns (uint256) {
+        uint8 baseDecimals = baseAddress == 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE ? 18 : ERC20Detailed(baseAddress).decimals();
+        PairAggregatorInterface aggregator = _getAggregatorFor(baseAddress, quoteAddress);
+        uint256 oneTokenPrice = uint256(aggregator.getLatestAnswer());
+        // TODO: better way with SafeMath?
```

Comment:
Is this TODO fixed?

Reply:
No I wasn't sure since there is no `mul` function in SafeMath.